### ISSUE TITLE
beta / 9.0.2 – Unified Interface: Clicking on a button with a submenu does not work

### ISSUE THREAD

**Issue by NicolasPrats** [Microsoft/EasyRepro#292]
Hello,

The method ```Microsoft.Dynamics365.UIAutomation.Api.UCI. CommandBar.ClickCommand()``` does not work as expected when the parameter ```submenu``` is provided. The exception ```"No command with the name '{name}' exists inside of Commandbar."``` It's thrown even if the command exists.
The equivalent method for the webfresh (```Microsoft.Dynamics365.UIAutomation.Api.XrmPageClickCommand.ClickCommand```) is working fine.

If it can help you, the issue seems related to the line: 
```C#
button = subname == "" ? items.FirstOrDefault(x => x.GetAttribute("aria-label") == name) : items.FirstOrDefault(x => x.GetAttribute("aria-label") == name + " More Commands");
```

If I replace this line with the following one, the button is working as expected in my case.
```C#
button = items.FirstOrDefault(x => x.GetAttribute("aria-label") == name);
```

**Comment by TYLEROL** [comment]
Thanks @NicolasPrats . I've added this to our current backlog and will have it reviewed as part of the v9.1 update work currently ongoing.

Regards,
Tyler

**Comment by ggeiszle** [comment]
Hey @NicolasPrats  - thanks for bringing this to our attention.  We addressed this issue in the 9.1 branch.  if you continue to see issues, please let us know!
